examples/ads.amp.html

```diff
@@ -557,6 +558,17 @@ <h2 id="ezoic">Ezoic</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="felmat">felmat</h2>
+  <amp-ad width=300 height=250
+      type="felmat"
+      data-host="felmat.net"
+      data-fmt="banner"
+      data-fmk="U12473_n2cJD"
+      data-fmp="">
```

Comment:
I see you don't assign a value to `data-fmp`, but you do include it in your example. Is `data-fmp` a required parameters or is it an optional one?
If it's optional I think you can remove it from here and change your doc and `validateData()` function as well.